### ISSUE TITLE
Mail HELO localhost

### ISSUE THREAD

**Issue by TheEagle13** [dani-garcia/bitwarden_rs#1041]
### Subject of the issue
Emails being sent by the instance are often being blocked by corporate subscription spam filters because the HELO command is sending localhost as the sending mail server name. This is frowned upon and therefore usually listed as spam by many corporate filters.

### Your environment

* Bitwarden_rs version: 2.14.0 

* Install method: Docker

### Steps to reproduce
Look at a pcap from email traffic

### Expected behaviour
should list a server name

### Actual behaviour
lists localhost

**Comment by dani-garcia** [comment]
Checking the lettre configuration, there's an option to provide the HELO name, should I create an option for it, or is it enough to use the SMTP host as-is?

**Comment by TheEagle13** [comment]
Thanks for the quick response!
Preference would be to allow a configurable option, because another issue that would come up is if the server name presented doesn't match the reverse dns lookup of the IP (which in my case it wouldn't since its a docker container on a host with a different name and reverse dns.)

**Comment by dani-garcia** [comment]
Okay, option `HELO_NAME` is added now in the SMTP settings in the latest commit: https://github.com/dani-garcia/bitwarden_rs/commit/596c9b869185a86d7619024066c80a008102199e.

The docker images will still take some hours to build, but once done I'd appreciate if you can give it a try!

**Comment by TheEagle13** [comment]
That did the trick! Thank you so much!
